apps/web/src/server/lib/nunjucks-template-builders/summary-list-builder.js

```diff
@@ -79,6 +85,7 @@ function getHtmlTags(type, link) {
 	 * @type {Object<string, {startTag: string, endTag: string}>}
 	 */
 	let determineHtmlTag = {
+		raw: { startTag: '', endTag: '' },
```

Comment:
I think it may need a span here instead. I tried removing it and it broke something on LPAQ I can't remember what  though.

Reply:
Yes that's probably wise - updated